src/quo2/components/record_audio/record_audio/buttons/record_button_big.cljs

```diff
@@ -45,129 +45,131 @@
+           stop-animation (fn []
+                            (set-value opacity 0)
+                            (reanimated/cancel-animation scale)
+                            (set-value scale 1)
+                            (when @clear-timeout (js/clearTimeout @clear-timeout)))
+           start-y-animation (fn []
+                               (reset! record-button-at-initial-position? false)
+                               (reset! record-button-is-animating? true)
+                               (animate-easing translate-y -64 250)
+                               (animate-linear-with-delay icon-opacity 0 33.33 76.66)
+                               (js/setTimeout (fn []
+                                                (reset! record-button-is-animating? false)
+                                                (when-not @touch-active? (complete-animation)))
+                                              250))
+           reset-y-animation (fn []
+                               (animate-easing translate-y 0 300)
+                               (animate-linear icon-opacity 1 500)
+                               (js/setTimeout (fn []
+                                                (reset! record-button-at-initial-position? true))
+                                              500))
+           start-x-animation (fn []
+                               (reset! record-button-at-initial-position? false)
+                               (reset! record-button-is-animating? true)
+                               (animate-easing translate-x -64 250)
+                               (animate-linear-with-delay icon-opacity 0 33.33 76.66)
+                               (animate-linear red-overlay-opacity 1 33.33)
+                               (js/setTimeout (fn []
+                                                (reset! record-button-is-animating? false)
+                                                (when-not @touch-active? (complete-animation)))
+                                              250))
+           reset-x-animation (fn []
+                               (animate-easing translate-x 0 300)
+                               (animate-linear icon-opacity 1 500)
+                               (animate-linear red-overlay-opacity 0 100)
+                               (js/setTimeout (fn []
+                                                (reset! record-button-at-initial-position? true))
+                                              500))
+           start-x-y-animation (fn []
+                                 (reset! record-button-at-initial-position? false)
+                                 (reset! record-button-is-animating? true)
+                                 (animate-easing translate-y -44 200)
+                                 (animate-easing translate-x -44 200)
+                                 (animate-linear-with-delay icon-opacity 0 33.33 33.33)
+                                 (animate-linear gray-overlay-opacity 1 33.33)
```

Comment:
I see these helpers were imported using `refer`. So do we use `refer` or not? cc @ilmotta @flexsurfer

Reply:
i would avoid using refer, if we can't use the full name for an alias for some cases shorter version could be used, if we all are aware about it, like `ra/` for reanimated , we can discuss this

Reply:
We already have a decision for this point @OmarBasem, it's in the guidelines.

>Prefer `:as` instead of `:refer`. There are exceptions to this rule, e.g. the
test macros `deftest` and `is`, which are ubiquitous in the Clojure community.

Reply:
Removed `refer`'s :)